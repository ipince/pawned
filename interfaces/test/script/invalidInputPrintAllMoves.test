# Invalid input in PrintAllMoves command
StartNewGame human 0 human 0
PrintAllMoves in
PrintAllMoves in in
StartNewGame computer 0 computer 0
PrintAllMoves in
PrintAllMoves in in
StartNewGame human 0 computer 0
PrintAllMoves in
PrintAllMoves in in
StartNewGame computer 0 human 0
PrintAllMoves in
PrintAllMoves in in