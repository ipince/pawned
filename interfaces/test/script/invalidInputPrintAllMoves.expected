# Invalid input in PrintAllMoves command
New game started
Input error
Input error
New game started
Input error
Input error
New game started
Input error
Input error
New game started
Input error
Input error
